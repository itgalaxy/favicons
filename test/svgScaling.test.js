const favicons = require("../src");
const { logo_small_svg } = require("./util");

// Only one iconset is required that contains large enough images to compare the
// difference between the generated files.
const icons = {
  android: false,
  appleIcon: true,
  appleStartup: false,
  coast: false,
  favicons: false,
  firefox: false,
  windows: false,
  yandex: false,
};

test("should scale the SVG image properly", async () => {
  expect.assertions(1);
  const result = await favicons(logo_small_svg, { icons });

  result.images = result.images.filter(
    (image) => image.name === "apple-touch-icon-1024x1024.png"
  );

  await expect(result).toMatchFaviconsSnapshot();
});
