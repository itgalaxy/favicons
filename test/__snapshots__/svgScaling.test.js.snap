// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should scale the SVG image properly 2`] = `
Array [
  Object {
    "contents": null,
    "name": "apple-touch-icon-1024x1024.png",
  },
]
`;
